includes/models/model.llms.course.php

```diff
@@ -874,7 +874,9 @@ public function get_end_date() {
 
 	/**
 	 * Find the next uncompleted lesson in the course syllabus
-	 * @return int [Next uncompleted lesson]
+	 * @return int|false next uncompleted lesson id or false if all lessons are complete
+	 * @since [version] Returns false if all lessons are complete.
+	 * @version [version]
 	 */
 	public function get_next_uncompleted_lesson() {
```

Comment:
This (and a handful of other functions at the bottom of this class) should have been marked as deprecated but were "deprecated" before I became aware of docblocking as a way of life so I "deprecated" them by never adding inline docks and moving them to the bottom of the file. This was done back in 2016 and I've forgotten these functions exist.